api/src/main/resources/workbench-api.yaml

```diff
@@ -6311,18 +6311,18 @@ definitions:
       duaTypeEnum:
         "$ref": "#/definitions/DuaType"
         description: 'Type of Data Use Agreement Institution has signed, Master or restricted to a defined set of specific researchers'
-      emailAddresses:
-        type: array
+      tierEmailAddresses:
         description: A list of email addresses which can be validated as members of
-          this Institution
-        items:
-          type: string
-      emailDomains:
+          this Institution for each access tier.
         type: array
+        items:
+          "$ref": "#/definitions/TierEmailAddresses"
+      tierEmailDomains:
         description: A list of email domains which can be validated as members of
-          this Institution
+          this Institution for each access tier.
+        type: array
         items:
-          type: string
+          "$ref": "#/definitions/TierEmailDomains"
```

Comment:
I realize this would be a significant change to the PR, and I missed prior discussion you may have had with Joel, but I really think a different API structure would be cleaner here:


```
tierConfigs: [{
  accessTierShortName: "",
  eRACommonsRequired: false,
  membershipType: enum,
  emailAddresses: [""], //optional
  emailDomains: [""] // optional
}, ...]
```

Reply:
We never considered this structure, and there are not too much design discussion about this. The decision is being discussed and made in design doc. The idea is:

- DuaType into TierRequirement
- emailDomains to tierEmailDomains
- emailAccesses to tierEmailAccessTier
And do the extended wha we have  in DB and API to the new format, and still 1 to 1 mapping. 
I have to think about that. That may make UI cleaner. Not sure how to update the mapper and backend DB update... First glance is that may make the new institution tier Ui easier to write... But it is a significant change.

Reply:
Now it is reverted...